### PR TITLE
tweak protonmail

### LINKED ISSUE #99: ProtonMail needs better text
https://providers.delta.chat/protonmail only says "it's broken".  Better to say it doesn't support the standard e-mail server protocols (IMAP and SMTP) ... maybe also read up on what ProtonMail themselves say. (there might be chances to get it to work on Desktop through the PRotonMail bridge).

### PR DESCRIPTION
**Pull request by r10s**
targets #99

### CODE REVIEW

File: _providers/protonmail.md [hunk 1]
```diff
@@ -5,12 +5,11 @@ domains:
 - protonmail.com
 - protonmail.ch
 before_login_hint: |
-  Unfortunately, Protonmail doesn't offer IMAP, so Delta Chat can't communicate with it.
-  We suggest to use a different email provider.
-after_login_hint: |
-  Unfortunately, Protonmail doesn't offer IMAP, so Delta Chat can't communicate with it.
-  If you really want to use Delta Chat with Protonmail, you can check out their experimental IMAP bridge: https://protonmail.com/bridge/
+  Unfortunately, Protonmail doesn't offer IMAP, so you cannot log in with Delta Chat to Protonmail.
```
Comment (hpk42) [inline _providers/protonmail.md]:
```suggestion
  Protonmail does not offer the standard IMAP e-mail protocol, so you cannot log in with Delta Chat to Protonmail.
```

File: validation/index.js [hunk 2]
```diff
@@ -27,19 +27,19 @@ function test(fileContent) {
...
     } else if (json.status == "BROKEN") {
         // If status == BROKEN, does before_login_hint & after_login_hint exist?
         if (!json.before_login_hint || json.before_login_hint.trim() === "") {
-            throw new Error("Status is Broken, but 'before_login_hint' is missing")
+            throw new Error("Status is BROKEN, but 'before_login_hint' is missing")
         }
-        if (!json.after_login_hint || json.after_login_hint.trim() === "") {
-            throw new Error("Status is Broken, but 'after_login_hint' is missing")
+        if (markdown.trim() === "") {
+            throw new Error("Status is BROKEN, but website content is missing")
```
Comment (missytake) [inline validation/index.js]: We do need the after_login_hint for broken providers as well. As we don't forbid logging into broken providers, it's technically possible, that people might make it. We should still leave a reminder in their inbox, that we consider the provider broken. If they use a provider successfully, they can still open an issue in this repository to reopen the provider.

Comment (missytake) [reply validation/index.js]: But yes, requiring markdown when BROKEN is still a good idea.

Comment (r10s) [reply validation/index.js]: i would be fine with that, it was just that the existing after_login_hint is a bit weird, pointing the user to things he has already done and i had no good idea what to say instead,

@missytake feel free to add a commit to this pr and fix the text and the condition.

**Comment by r10s** [comment]
thinking it over, i do not think, we should make the `after_login_hint` really a MUST.

eg. this pr already pointed out that it is not that easy to figure out what to write there.  also, if you make it  a MUST, you have think about some wording that does never appear for the vast majority of the users - this time can better be spent on more low-hanging fruits.

**Comment by missytake** [comment]
Okay, I agree. I added an after_login_hint for protonmail, but let's not require it for every BROKEN provider.

merge?

**Comment by missytake** [comment]
Ah, it's up to me anyway. I'll merge it.

**Pull request merged**